.github/workflows/build.yml

```diff
@@ -61,10 +24,10 @@ jobs:
 
           docker build . \
             --file Dockerfile \
-            --target artifact \
+            --target vvgo \
             --cache-from=$CACHE_FROM:latest-builder \
             --cache-from=$CACHE_FROM:latest \
-            --tag artifact
+            --tag vvgo
```

Comment:
Don't really see a reason to change this: `vvgo:vvgo` isn't very semantic. I'd rather keep artifact.

Reply:
Atifact is too... generic. What about webapp or something like that?

Reply:
Eh actually doesn't matter

Reply:
Actually, what is this tag even used for?

Reply:
Oh I think this should be the branch name of the pr?

Reply:
The tag is used for labeling Docker images. I'd probably tag it `artifact-$branch`, to avoid conflicts and overwriting a release.

Reply:
I guess the question is, how do we plan to use the tags?

Reply:
Ah it looks like we never actually push that tag right?

Reply:
We push tags $IMAGE_ID:$VERSION but not these tags.

Reply:
Thoughts:

- `latest`
- `$version`
- `pr-$branch`

Reply:
> Ah it looks like we never actually push that tag right?
Yeah, that's true.